include/chemfiles/formats/TNG.hpp

```diff
@@ -27,6 +27,8 @@ class TNGFormat final: public Format {
 
     /// Reference to the associated file
     TNGFile tng_;
+    /// Scale factor for all lenght dependent data e.g. positions, velocities, forces, box shape
+    float distanceScaleFactor_ = -1;
```

Comment:
Could you use the `snake_case` instead? This is the main convention in chemfiles =)

Reply:
Sure, done.